src/layouts/docs/docs-layout.module.css

```diff
@@ -46,6 +46,7 @@
 .main {
   max-width: 804px;
   padding: 0 24px;
+  padding-bottom: 50vh;
```

Comment:
I see the comment about the reasoning to have this but this much empty space makes it feel like something isn't rendering as expected here.

Reply:
Hadn't thought about that. My personal reasons for liking it is because I like having content at the top of my screen when reading. But I'd really like to hear from more people on this and if they've seen this addressed in some other way!

Reply:
This is a fair point by Alex. I think it's probably worth raising to the broader group. If we do want to keep it around, maybe there's some way we can make it feel more intentional 💭

Reply:
I also remembered this morning that there will be a footer component added directly below the content/sidecar portion of the page, I wonder if this will help alleviate what we're observing with empty space? Link to beta footer design here: https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=2835%3A60851